### PR TITLE
Version 4.16.0

### PR DESCRIPTION
**Pull request by kumavis**

### CODE REVIEW

**Review by 2-am-zzz — DISMISSED** [review]
On the 16th day of DevCon my full node gave to me...

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,19 +2,37 @@
 
 ## Current Develop Branch
 
+## 4.16.0 Thursday October 17 2018
```
Comment (whymarrh) [inline CHANGELOG.md]: Nit: we should update the date

**Review by whymarrh — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (Zanibas) [reply CHANGELOG.md]: Isn't it still October 17?

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,19 +2,37 @@
...
+- Feature: add tooltip for view etherscan tx
+- Feature: add Polish translations
+- Feature: improve Korean translations
+- Feature: improve Italian translations
+- Bug Fix: Fix bug with "pending" block reference
+- Bug Fix: Force AccountTracker to update balances on network change
+- Bug Fix: Fix document extension check when injecting web3
+- Bug Fix: Fix some support links
+
+## 4.15.0 Thursday October 17 2018
```
Comment (whymarrh) [inline CHANGELOG.md]: Also this isn't a date

**Review by whymarrh — DISMISSED** [review]

**Review by whymarrh — APPROVED** [review]
All LGTM (including the changelog dates) 👍

**Pull request merged**